server/src/services/generatorFlow.js

```diff
@@ -0,0 +1,188 @@
+/* eslint-disable no-plusplus */
+/* eslint-disable consistent-return */
+import { generator } from './generator';
+import { channelMessageValidator, isJson } from '../utils/schemaValidators';
+import schemaCreator from '../utils/schemaCreator';
+import defaultSpec from '../models/defaultSpec.json';
+import questions from '../models/questions';
+import { childEntityValue } from '../utils/entities';
+
+const document = defaultSpec;
+
+export default function generatorFlow(
+  entities,
+  socket,
+  io,
+  c,
+  b,
```

Comment:
You should name the variables as `a`, `b` or something similar. To be honest, it's a hard to understand what contain these variables. Could you change to something more verbose?

Reply:
Ok, it's a POC so @AceTheCreator, you can leave as it is and fix it in the next PRs.